pkgs/games/natural-pong/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, udev, alsa-lib, vulkan-loader, xorg, libxkbcommon, wayland, pkg-config, makeWrapper, rustPlatform, fetchFromGitHub }:
+let
+  buildInputs = [
+    udev alsa-lib vulkan-loader
+    xorg.libX11 xorg.libXcursor xorg.libXrandr xorg.libXi
+    libxkbcommon wayland
+  ];
+  nativeBuildInputs = [
+    pkg-config
+    makeWrapper
+  ];
+in rustPlatform.buildRustPackage {
+  pname = "natural-pong";
+  version = "0.1.0";
+  cargoHash = "sha256-jfeQjZrGyGXQ298naakiDpSlbl116ZKyi0RBkLnlwsA=";
+
+  buildInputs = buildInputs;
+  nativeBuildInputs = nativeBuildInputs;
+  LD_LIBRARY_PATH = lib.makeLibraryPath buildInputs;
+  src = fetchFromGitHub {
+    owner = "M0rious";
+    repo = "3d-pong";
+    rev = "3cbe82e080bf894622364a88322fc19b7b5fc447";
+    hash = "sha256-0gQO5PX9vVcsGf3z2ikWtCNtuC1b855bLXMsvJRU23o=";
+  };
+  postInstall = ''
+    mv $out/bin/pong $out/bin/natural-pong
```

Comment:
that dons't look quite right but to be honest I don't know anything better.

Reply:
It should be a symlink.